scripts/analytics/institutions.py

```diff
@@ -0,0 +1,86 @@
+from modularodm import Q
+
+from website.app import init_app
+from website.models import User, Node, Institution
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+
+def get_institutions():
+    institutions = Institution.find(Q('_id', 'ne', None))
+    return institutions
+
+
+def get_count_by_institutions():
+    institutions = get_institutions()
+    counts = []
+
+    for institution in institutions:
+        user_query = Q('_affiliated_institutions', 'eq', institution.node)
+        node_query = (
+            Q('is_deleted', 'ne', True) &
+            Q('is_folder', 'ne', True)
+        )
+        registration_query = node_query & Q('is_registration', 'eq', True)
+        non_registration_query = node_query & Q('is_registration', 'eq', False)
+        project_query = non_registration_query & Q('parent_node', 'eq', None)
+        registered_project_query = non_registration_query & Q('parent_node', 'eq', None)
+        public_query = Q('is_public', 'eq', True)
+        private_query = Q('is_public', 'eq', False)
+        node_public_query = non_registration_query & public_query
+        node_private_query = non_registration_query & private_query
+        project_public_query = non_registration_query & public_query
+        project_private_query = non_registration_query & private_query
+        registered_node_public_query = registration_query & public_query
+        registered_node_private_query = registration_query & private_query
+        registered_project_public_query = registered_project_query & public_query
+        registered_project_private_query = registered_project_query & private_query
+        count = {
+            'id': institution._id,
+            'institution': institution.name,
```

Comment:
Instead of `id` and `institution` as same level keys, group them together:
````
{
  'institution': {
    'id': institution._id,
    'name': institution.name,
  }
}
````
Later we might expand what we capture about the institutions (e.g. type: higher_ed|professional|nonprofit) and grouping properties makes reading queries more intuitive.